libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -429,6 +429,17 @@ bool AC_Fence::pre_arm_check(char *failure_msg, const uint8_t failure_msg_len) c
         hal.util->snprintf(failure_msg, failure_msg_len, "Fences enabled, but none selected");
         return false;
     }
+    
+    // if AUTOENABLE = 1 or 2 warn now, but fail in a later release
+    // PARAMETER_CONVERSION - Added: Jul-2024 for ArduPilot-4.6
+    const uint32_t now_ms = AP_HAL::millis();
+    static uint32_t _last_autoenable_warn_ms = now_ms - 60000; //needs this static since function is const
```

Comment:
These timings seem fairly arbitrary and pre-arm already has its own timings for output. I would just make this a one-line message and print every time.

Reply:
its not a pre-arm yet and we dont want to spam at 30 second rate, nor to make it a one off thats missed